evm-contracts/app.config.ovm.json

```diff
@@ -0,0 +1,16 @@
+{
+  "contractsDir": "src",
+  "artifactsDir": "abi",
+  "contractAbstractionDir": ".",
+  "useDockerisedSolc": false,
+  "compilerSettings": {
+    "optimizer": {
+      "runs": 1000000,
```

Comment:
Do we need a high number of runs on Optimism? Is there any benefit to optimizing contract size for the verifiers?

Reply:
Yes, you are correct! I took that specific number from somewhere, but can't remember where anymore.

We are optimizing for contract size on OVM L2, so we should set the optimizer to only 1 run.

I just tested this today and both `AggregatorProxy.test.ts` & `FluxAggregator.test.ts` will get an "out of gas" error for 1000000 runs, but they seem to work fine when the compiler optimizer is configured to only do 1 run.

Reply:
Nice, let's drop it.